flink-yarn/src/main/java/org/apache/flink/yarn/cli/FlinkYarnSessionCli.java

```diff
@@ -439,8 +450,8 @@ public static void runInteractiveCli(YarnClusterClient yarnCluster, boolean read
 						case "quit":
 						case "stop":
 							yarnCluster.shutdownCluster();
+							if (yarnCluster.hasBeenShutdown()) { removeAppState(applicationId); }
```

Comment:
This is not consistent with the style in the rest of the Flink codebase. Can you put the "remoteAppState()" into a separate line?